README.md

```diff
@@ -1 +1,89 @@
+* [Kitura](https://www.kitura.io): Kitura is a free and open-source web framework written in Swift, developed by IBM and licensed under Apache 2.0. It’s an HTTP server and web framework for writing Swift server applications.
+
+## Featured Technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
+* [Swift on the Server](https://developer.ibm.com/swift/): Build powerful, fast and secure server side Swift apps for the Cloud.
+
+# Prerequisites
+
+* If you don't already have a PowerAI server, you can acquire one from [Nimbix](https://www.nimbix.net/ibm) or from the [PowerAI offering](https://console.bluemix.net/catalog/services/powerai) on IBM Cloud.
+* macOS 10.13 (High Sierra) or later
+
+# Steps
+
+1. [Clone the repo](#1-clone-the-repo)
+1. [Download the LaMem dataset](#2-download-the-lamem-dataset)
+1. [Train the Keras model](#3-train-the-keras-model)
+1. [Convert the Keras model to a CoreML model](#4-convert-the-keras-model-to-a-coreml-model)
+1. [Run the Kitura web app](#5-run-the-kitura-web-app)
+
+### 1. Clone the repo
+
+Clone the `powerai-image-memorability` repo onto both your PowerAI server and local macOS machine. In a terminal, run:
+
+```
+git clone https://www.github.com/IBM/powerai-image-memorability
+```
+
+### 2. Download the LaMem data
+
+To simplify the data download and extraction procedure, this code pattern contains a script that you can run to download and extract the data for you. To run this script, `cd` into the `powerai_serverside` directory in a terminal:
+
+```
+cd powerai_serverside
+```
+
+Then, run the following command to execute the script:
+
+```
+sh download_extract_data.sh
+```
+
+### 3. Train the Keras model
+
+To train the Keras model, run the following command inside of the `powerai_serverside` directory:
+
+```
+python train_nasnet.py
+```
+
+Once Python script is done running, you'll see a `current_best.hdf5` model in the `powerai_serverside` directory. Copy that over to the `webapp` directory on the macOS machine that you'd like to run the frontend on.
+
+### 4. Convert the Keras model to a CoreML model
+
+Inside of the `webapp` directory on your macOS machine, run the following Python script to convert your Keras model to a CoreML model:
+
+```
+python convert_model.py current_best.hdf5
+```
+
+This may take a few minutes, but when you're done, you should see a `lamem.mlmodel` file in the `webapp` directory.
+
+### 5. Run the Kitura web app
+
+Then, you're ready to roll! Run the following command to build & run your application:
+
+```
+swift build && swift run
+```
+
+Now, you can head over to `localhost:3333` in your favourite web browser, upload an image, and calculate its memorability.
```

Comment:
Include a screenshot of the running application

Reply:
Also include Links (code attributions), troubleshooting (if you can think of something), and license.